src/ResourceManager/Network/Commands.Network/ApplicationGateway/FirewallConfiguration/AzureApplicationGatewayWebApplicationFirewallConfigurationBase.cs

```diff
@@ -49,8 +49,33 @@ public class AzureApplicationGatewayWebApplicationFirewallConfigurationBase : Ne
         [Parameter(
                Mandatory = false,
                HelpMessage = "The disabled rule groups.")]
+        [Alias(DisabledRuleGroupsAlias)]
         [ValidateNotNullOrEmpty]
-        public List<PSApplicationGatewayFirewallDisabledRuleGroup> DisabledRuleGroups { get; set; }
+        public List<PSApplicationGatewayFirewallDisabledRuleGroup> DisabledRuleGroup { get; set; }
+
+        [Parameter(
```

Comment:
you need to generate the help files for the cmdlets that this base class is used in as well.. as we added a  few params here.